src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -61,11 +61,20 @@ internal bool NeedsGeneratedIsByRefLikeAttribute
         {
             get
             {
-                _needsGeneratedIsByRefLikeAttribute_IsFrozen = true;
+                _needsGeneratedAttributes_IsFrozen = true;
                 return _needsGeneratedIsByRefLikeAttribute_Value;
             }
         }
 
+        internal bool NeedsGeneratedNullableAttribute
+        {
+            get
+            {
+                _needsGeneratedAttributes_IsFrozen = true;
```

Comment:
>_needsGeneratedAttributes_IsFrozen [](start = 16, length = 34)

Perhaps we should assert that this field is false on entry for all three properties.

Reply:
Wait, we cannot do this now since we are sharing it.

---
In reply to: [147518519](https://github.com/dotnet/roslyn/pull/22820#discussion_r147518519) [](ancestors = 147518519)